app/services/tag_service.rb

```diff
@@ -0,0 +1,17 @@
+class TagService
```

Comment:
```def tag_list``` powędrowało później do presentera. 
czy ```def set_tag_list``` może być w serwisie jako metoda wywoływana w kontrolerze todo_item ale związana m.in. z tworzeniem tagów dla item'a? czy też powinna gdzieś powędrować?